cmd/applier/main.go

```diff
@@ -24,17 +25,23 @@ func main() {
 	var kubeconfigPath string
 	var dryRun bool
 	var prefix string
+	var delete bool
+	var timeout int
+	var force bool
 	klog.InitFlags(nil)
 	flag.StringVar(&dir, "d", ".", "The directory containing the templates, default '.'")
 	flag.StringVar(&valuesPath, "values", "values.yaml", "The directory containing the templates, default 'values.yaml'")
 	flag.StringVar(&kubeconfigPath, "k", "", "The kubeconfig file")
 	flag.BoolVar(&dryRun, "dry-run", false, "if set only the rendered yaml will be shown, default false")
 	flag.StringVar(&prefix, "p", "", "The prefix to add to each value names, for example 'Values'")
+	flag.BoolVar(&delete, "delete", false, "if set only the resource defined in the yamls will be deleted, default false")
+	flag.IntVar(&timeout, "t", 5, "Timeout in second to apply one resource, default 1 sec")
```

Comment:
default 5 second?

Reply:
5 sec per transaction (CRUD). You think too much or too little